README.md

```diff
@@ -1,15 +1,12 @@
 # Dynatrace OpenTelemetry Metrics Exporter for Java
 
-This exporter allows exporting metrics created using the [OpenTelemetry SDK for Java](https://github.com/open-telemetry/opentelemetry-java)
-directly to [Dynatrace](https://www.dynatrace.com).
+This exporter allows exporting metrics created using the [OpenTelemetry SDK for Java](https://github.com/open-telemetry/opentelemetry-java) directly to [Dynatrace](https://www.dynatrace.com).
+
+It was built against OpenTelemetry SDK version [1.14.0](https://github.com/open-telemetry/opentelemetry-java/releases/tag/v1.14.0) and should work with any 1.14.+ version.
```

Comment:
```suggestion
It was built against OpenTelemetry SDK version [1.14.0](https://github.com/open-telemetry/opentelemetry-java/releases/tag/v1.14.0) and should work with any 1.14+ version.
```
(can be ignored, just am not sure if we should keep the `.`) :slightly_smiling_face:

Reply:
Sounds good, it's just a nit anyway. :slightly_smiling_face: 
I thought that I usually see that written as `1.14+`, but could have been my brain fooling me. :sweat_smile:

Reply:
oh 😮, you are right. I thought I had it right. Really nice catch, updating now.